newsletter.css

```diff
@@ -279,23 +256,23 @@ a {
     font-weight: normal;
     font-size: 16px;
     letter-spacing: 0.02em;
-    text-align: center;
 
-    margin-left: 10px;
+    margin-left: 7px;
+
```

Comment:
not that I'm looking _super_ hard at css files, but there's a lot of weird whitespaces everywhere

Reply:
Ah yes I thought the whitespaces helped separate the sections but technically that's redundant since I have comments. Thanks!